src/test/java/plugins/RemotingPluginTest.java

```diff
@@ -0,0 +1,61 @@
+package plugins;
+
+import hudson.util.VersionNumber;
+import org.apache.commons.io.FileUtils;
+import org.jenkinsci.test.acceptance.junit.AbstractJUnitTest;
+import org.jenkinsci.test.acceptance.junit.WithPlugins;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.jar.Attributes;
+import java.util.jar.JarFile;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+
+/**
+ * @author Kohsuke Kawaguchi
+ */
+public class RemotingPluginTest extends AbstractJUnitTest {
+    @Rule
+    public TemporaryFolder tmp = new TemporaryFolder();
+
+    /**
+     * When installing a core component override plugin, we should see that plugin functioning.
+     *
+     * <p>
+     * Run with environment variable: <tt>REMOTING_JPI=path/to/your-locally-build-remoting.hpi
+     * <p>
+     * We check this by:
+     * <ol>
+     *   <li>Testing the version of the plugin as declared installed
+     *   <li>Making sure that the actual jar file in the system is replaced
+     *  </ol>
+     */
+    @Test
+    @WithPlugins({"remoting"})
+    @Ignore // meant to be used to locally demonstrate JENKINS-41196 support
```

Comment:
I think you can omit that, since it will not run the test unless the plugin is available anyway.

Reply:
No, the test will fail saying that the said plugin is not available in the update center, as it should be. If you made a typo in a plugin name you'd want the test to fail to tell you something is wrong, instead of ignoring a test.